tests/custom_migration_operations/operations.py

```diff
@@ -68,6 +68,24 @@ def deconstruct(self):
         )
 
 
+class KeywordOnlyArgsOperation(TestOperation):
+    def __init__(self, *, kwarg1, kwarg2):
+        self.kwarg1 = kwarg1
+        self.kwarg2 = kwarg2
+
+    def deconstruct(self):
+        kwargs = {}
+        if self.kwarg1 is not None:
+            kwargs["kwarg1"] = self.kwarg1
+        if self.kwarg2 is not None:
+            kwargs["kwarg2"] = self.kwarg2
+        return (
+            self.__class__.__name__,
+            [],
+            kwargs,
+        )
```

Comment:
Can we add positional-only arguments to cover all changes?

Reply:
Yup I was actually thinking the same thing after signing off the day… great minds think alike. 😜

Reply:
Ah sorry we can't do positional-only args for operations because they're serialised as kwargs.

We can do positional-or-keyword-args + keyword-only args in this test… which is what I was thinking we could add.

Reply:
Good point :+1: